src/cryptoadvance/specter/server_endpoints/controller.py

```diff
@@ -137,3 +139,54 @@ def get_scantxoutset_status():
         "active": app.specter.info["utxorescan"] is not None,
         "progress": app.specter.info["utxorescan"],
     }
+
+
+@app.route("/bitcoin.pdf")
+@login_required
+def get_whitepaper():
+    if app.specter.chain == "main":
+        if not app.specter.info["pruned"]:
+            raw_tx = app.specter.rpc.getrawtransaction(
+                "54e48e5f5c656b26c3bca14a8c95aa583d07ebe84dde3b7dd4a78f4e4186e713",
+                False,
+                "00000000000000ecbbff6bafb7efa2f7df05b227d5c73dca8f2635af32a2e949",
+            )
+            outputs = raw_tx.split("0100000000000000")
+            pdf = ""
+            for output in outputs[1:-2]:
+                cur = 6
+                pdf += output[cur : cur + 130]
+                cur += 132
+                pdf += output[cur : cur + 130]
+                cur += 132
+                pdf += output[cur : cur + 130]
+            pdf += outputs[-2][6:-4]
+        else:
+            outputs_prun = app.specter.rpc.multi(
```

Comment:
@ben-kaufman and here is the rpc.multi implementation for pruned nodes, much faster!